priv/templates/nodetool

```diff
@@ -32,17 +32,10 @@ main(Args) ->
             io:format("~p\n", [rpc:call(TargetNode, init, restart, [], 60000)]);
         ["reboot"] ->
             io:format("~p\n", [rpc:call(TargetNode, init, reboot, [], 60000)]);
+        ["rpc", Module, Function | RpcArgs] when RpcArgs == [] ->
```

Comment:
this can just be `["rpc", Module, Function]` since `[]` would mean simply the end of the list.